.github/workflows/nodejs.yml

```
@@ -0,0 +1,29 @@
+name: CI
+
+on: [push]
```

You also must use `pull_request`. If you don't folks that PR this repo from their own forks will not run CI.

My suggested setup:

```suggestion
on:
  pull_request:
  push:
    # filtering branches here, prevents duplicate builds from pull_request and push
    branches: 
      - master
      - 'v*'
    # always run CI for tags
    tags:
      - '*'
      
  # early issue detection, run CI weekly on Sundays  
  schedule:
  - cron:  '0 6 * * 0' 
```